account_invoice_supplierinfo_update_discount/tests/test_module.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2018 - Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from datetime import datetime
+from odoo.tests.common import TransactionCase
+
+
+class TestModule(TransactionCase):
+
+    def setUp(self):
+        super().setUp()
+        self.AccountInvoice = self.env['account.invoice']
+        self.WizardUpdate = self.env['wizard.update.invoice.supplierinfo']
+        self.SupplierInfo = self.env['product.supplierinfo']
+
+        self.product1 = self.env.ref('product.product_product_4b')
+        unit = self.env.ref('uom.product_uom_unit')
+        account_id = self.env['account.account'].search(
+            [('user_type_id.type', '=', 'payable')], limit=1).id
+        journal_id = self.env['account.journal'].search(
+            [('type', '=', 'sale')], limit=1).id
```

Comment:
@legalsylvain , the type of journal is not 'purchase' for the purchase invoice?

Reply:
Hi @chafique-delli. It doesn't change nothing, but indeed, it is not clean. 

I fixed it.

thanks !